metrics/default_metrics/luajit/init.lua

```diff
@@ -0,0 +1,75 @@
+local metrics = require('metrics')
```

Comment:
please add this file to rpm spec and rockspec

Reply:
Fixed